apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts

```diff
@@ -197,18 +190,13 @@ export class ProjectsListComponent implements OnInit {
         const uuid = this._projectService.iriToUuid(id);
 
         // the deleteProject() method in js-lib sets the project's status to false, it is not actually deleted
-        this._dspApiConnection.admin.projectsEndpoint
+        this.deactivateProjectSub = this._dspApiConnection.admin.projectsEndpoint
```

Comment:
I know and I also prefer that. But it is not in the scope of this PR to fix this imho. I only wanted to unsubscribe for the moment, because it is just bad. I don't touch it next time

Reply:
Well since you've already touched it, let's make it better :) You've added two subscription variables and an ngOnDestroy method to do what a .pipe(take(1)) can do. 

See this example:
```
this._advancedSearchService
                    .filteredPropertiesList(
                        property.selectedProperty?.objectType
                    )
                    .pipe(take(1))
                    .subscribe((properties) => {
                        property.childPropertiesList = properties;
                        property.searchValue = [];
                        this.updatePropertyFormListItem(
                            currentPropertyFormList,
                            property,
                            index
                        );
                    });
```

Reply:
I am super glad to change it, of course. I only tried to rescue stuff which come up on my way - but at the same time I like to keep boundaries of what is the purpose of the PR. So next time huge PR's from me ;-)